index.ts

```diff
@@ -29,7 +29,7 @@ class ServerlessCloudWatchLogsTagPlugin {
   }
 
   private get stackName(): string {
-    return `${this.serverless.service['service']}-${this.options.stage}`;
+    return this.awsService.naming.getStackName();
```

Comment:
I guess we can do a little bit shorter...
```suggestion
    return this.awsService.getStackName();
```
Didn't test this suggestion : it's only based on type found in `@types/serverless/plugins/aws/provider/awsProvider.d.ts`:
```typescript
declare class Aws {
    constructor(serverless: Serverless, options: Serverless.Options)

    naming: { [key: string]: () => string };
    getProviderName(): string;
    getRegion(): string;
    getServerlessDeploymentBucketName(): string;
    getStage(): string;
}
```

Reply:
I don't think it would work, there isn't a getStackName() in the AWS class (but I can test tomorrow)

Reply:
Fine. Please keep me posted with the result, I'll approve it as soon as received.

Reply:
Hello again, so I tested with the 'naming' and got the following (truncated) log

```
Serverless: Tagged LogGroup /aws/lambda/... with the following tags: ...
```

Without the naming I get a

```
TypeError: this.awsService.getStackName is not a function
```